src_new/adapters/prebid.js

```diff
@@ -592,6 +600,17 @@ function fetchBids(activeSlots, impressionID){
 
 				}
 
+				if(isRubiconPresent){
```

Comment:
can be moved to a common function function

Reply:
I introduced a loop which adds config to selected sra enable bidders